### DISCUSSION TITLE
RavenDB with Asp.Identity, Identity Server 4, Finbuckle Multitenant

### DISCUSSION THREAD

**Discussion by mcrio** [ravendb/ravendb#12636] [Show and tell]
Hi,

For those who might need it, I uploaded to Github some of my libraries which cover `Asp.Identity`, `IdentityServer4` and `FinbuckleMultitenant` with RavenDb stores. Those can be extended to fit a *multi-tenant use case.

I know there were already some libraries covering these topics, but as described in the Readme, ended up writing a new implementation to meet my requirements. 

RavenDB & Asp.net Core Identity
https://github.com/mcrio/Mcrio.AspNetCore.Identity.On.RavenDb

RavenDb & IdentityServer4
https://github.com/mcrio/Mcrio.IdentityServer.On.RavenDb

RavenDb & Finbuckle.Multitenant
https://github.com/mcrio/Mcrio.Finbuckle.MultiTenant.RavenDb.Store

(*) for Identity and IdentityServer see guidelines in readme regarding possible multi-tenant usage.

**Comment by ppekrol** [comment]
@mcrio 

Thank you for this fantastic contribution. We will add those to our [Awesome RavenDB](https://github.com/ravendb/awesome-ravendb) repository soon.

We planned to release our [IdentityServer4](https://github.com/ravendb/IdentityServer4.Contrib.RavenDB) package soon, but there might be absolutely no need to do so as I can see.

**Reply by mcrio** [reply]
Great, it's always great to have multiple options to choose from  👍

**Reply by DejanMilicic** [reply]
@mcrio Your contributions are in the Awesome repo. Thanx for your effort! 🙏
